### PR TITLE
feat: [M3-7108] - Add AGLB Configuration Delete Dialog

### PR DESCRIPTION
**Pull request by bnussman-akamai**
## Description 📝

- Adds AGLB Configuration delete dialog 🗑️

## Preview 📷

Screenshot 2023-09-13 at 4 05 38 PM

## How to test 🧪
- Turn on the MSW
- Go to `http://localhost:3000/loadbalancers/0/configurations`
- Try deleting a Configuration
  - Check the dialog for spelling, styles, and functionality
  - ⚠️ Because of how the MSW and React Query work, the configuration delete action might seem a bit weird

### CODE REVIEW

File: packages/manager/src/features/LoadBalancers/LoadBalancerDetail/Configurations/DeleteConfigurationDialog.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+            onClick: onClose,
+          }}
+        />
+      }
+      error={error?.[0]?.reason}
+      onClose={onClose}
+      open={open}
+      title={`Delete Configuration ${configuration.label}?`}
+    >
+      Are you sure you want to delete this configuration?
```
Comment (carrillo-erik) [inline packages/manager/src/features/LoadBalancers/LoadBalancerDetail/Configurations/DeleteConfigurationDialog.tsx:53]: Should this string be wrapped in a `Typography` component?

**Comment by carrillo-erik** [comment]
Checked spelling, styles, and functionality on the dialog and everything looks good. My single reservation is around the usage of a `Typography` component mentioned my comment.

**Review by carrillo-erik — APPROVED** [review]

**Review by cpathipa — APPROVED** [review]
Nice work! Configuration Delete Dialog works as expected.

File: packages/manager/src/features/LoadBalancers/LoadBalancerDetail/Configurations/DeleteConfigurationDialog.tsx (hunk 1 above)
Comment (bnussman-akamai) [reply packages/manager/src/features/LoadBalancers/LoadBalancerDetail/Configurations/DeleteConfigurationDialog.tsx:53]: I see no visual difference when using a `Typography`. I think we don't need to use one anymore 🎉

**Pull request merged**